FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2PersonaListActivity.kt

```diff
@@ -227,35 +227,20 @@ class V2PersonaListActivity : DemoActivity() {
                 title = "Elvia Atkins",
                 subTitle = "Intern",
                 trailingIcon = { trailingIcon() },
-                onClick = { onClick("Elvia Atkins", coroutineScope, scaffoldState) })
+                onClick = { onClick("Elvia Atkins", context) })
         )
     }
 
-    @SuppressLint("UnusedMaterialScaffoldPaddingParameter")
     @Composable
-    fun createActivityUI() {
-        val scaffoldState: ScaffoldState = rememberScaffoldState()
-        val coroutineScope: CoroutineScope = rememberCoroutineScope()
-        Scaffold(scaffoldState = scaffoldState) {
-            PersonaList(
-                modifier = Modifier.testTag("list"),
-                personas = createPersonasList(scaffoldState, coroutineScope),
-                border = BorderType.Bottom,
-                borderInset = BorderInset.XXLarge
-            )
-        }
+    fun createPersonaListActivityUI(context:Context) {
+        PersonaList(
+            personas = createPersonasList(context),
+            border = BorderType.Bottom,
+            borderInset = BorderInset.XXLarge
+        )
     }
 
-    fun onClick(text: String, coroutineScope: CoroutineScope, scaffoldState: ScaffoldState) {
-        scaffoldState.snackbarHostState.currentSnackbarData?.dismiss()
-        coroutineScope.launch {
-            val result = scaffoldState.snackbarHostState.showSnackbar(
-                message = "Clicked on $text",
-                actionLabel = "Close"
-            )
-            when (result) {
-                SnackbarResult.ActionPerformed -> scaffoldState.snackbarHostState.currentSnackbarData?.dismiss()
-            }
-        }
+    fun onClick(text: String, context: Context) {
```

Comment:
This function has been added in DemoUtils. Remove this & refer the DemoUtils one.
Code Pointer :
https://github.com/microsoft/fluentui-android/blob/master/FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/util/DemoUtils.kt#L22